services/FeefoReviewService.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Craft;
+
+class FeefoReviewService extends BaseApplicationComponent
+{
+  public $plugin = null;
+  public $settings = array();
+
+  public function __construct()
+  {
+    $this->plugin   = craft()->plugins->getPlugin('feeforeview');
+    $this->settings = $this->plugin->getSettings();
+  }
+
+  public function getReviewSummary($id)
+  {
+    if ($this->feefoMerchantId() == null) return null;
+
+    $products = $this->getProducts();
+    if ($products == null) return null;
+
+    $key = array_search($id, array_column($products, 'vendor_ref'));
```

Comment:
Although this is not wrong it may be confusing for others in the future that this variable is called $key since array_column with 2 parameters will return an indexed array and array_search will return the index that matches $id.
Calling this $index instead of $key will probably cause less confusion to someone unfamiliar with the code

Reply:
Correct. I changed it to $index.